subworkflows/local/prepare_references/main.nf

```diff
@@ -0,0 +1,105 @@
+include { SAMTOOLS_BGZIP } from '../../../modules/nf-core/samtools/bgzip/main'
+include { SAMTOOLS_FAIDX } from '../../../modules/nf-core/samtools/faidx'
+include { BWAMEM2_INDEX  } from '../../../modules/nf-core/bwamem2/index'
+include { MINIMAP2_INDEX } from '../../../modules/nf-core/minimap2/index'
+
+workflow PREPARE_REFERENCES {
+    take:
+    ch_samplesheet_reads_srt
+    ch_samplesheet_reads_lng
+    ch_genome
+
+    main:
+    // general order of precedence: reference_path -> param -> default
+    def reference_path = params.reference_path
+    def ch_versions = Channel.empty()
+
+    // prepare fasta
+    def defaultFasta = [
```

Comment:
Is this in readme? what we use as base reference?

Reply:
Good catch, they are out of date in the README: https://github.com/BfArM-MVH/GRZ_QC_Workflow?tab=readme-ov-file#running-the-pipeline-offline

Reply:
that would be great if you can also update those info